sdk/metric/config.go

```diff
@@ -140,3 +149,35 @@ func WithView(views ...View) Option {
 		return cfg
 	})
 }
+
+// WithExemplarFilter configures the exemplar filter.
+//
+// The exemplar filter determines which measurements are offered to the
+// exemplar reservoir, but the exemplar reservoir makes the final decision of
+// whether to store an exemplar.
+//
+// By default, the [go.opentelemetry.io/otel/sdk/metric/exemplar.SampledFilter]
+// is used. Exemplars can be entirely disabled by providing the
+// [go.opentelemetry.io/otel/sdk/metric/exemplar.AlwaysOffFilter].
```

Comment:
nit: given `go.opentelemetry.io/otel/sdk/metric/exemplar` is imported already:

```suggestion
// By default, the [exemplar.SampledFilter]
// is used. Exemplars can be entirely disabled by providing the
// [exemplar.AlwaysOffFilter].
```